examples/android_app/app/src/main/java/com/example/llamastackandroiddemo/LlamaStackLocalInference.kt

```diff
@@ -0,0 +1,148 @@
+package com.example.llamastackandroiddemo
+
+import android.util.Log
+import com.llama.llamastack.client.LlamaStackClientClient
+import com.llama.llamastack.client.local.LlamaStackClientLocalClient
+import com.llama.llamastack.core.JsonNumber
+import com.llama.llamastack.models.CompletionMessage
+import com.llama.llamastack.models.InferenceChatCompletionParams
+import com.llama.llamastack.models.SystemMessage
+import com.llama.llamastack.models.UserMessage
+import java.util.concurrent.CompletableFuture
+
+class LlamaStackLocalInference(
```

Comment:
I'd rename this to something like ExampleAppInferenceController (or similar) – at first glance I thought this is a provider implementation but it's the app-specific wrapper. Same with Remote.

Reply:
Make sense. Renamed to avoid confusion